tests/Phrasing/EmTest.php

```
@@ -0,0 +1,637 @@
+                )
+                ->render(),
+            'Event handler map must be applied.',
+        );
+    }
+
+    public function testRenderWithGlobalDefaultsAreApplied(): void
+    {
+        SimpleFactory::setDefaults(
+            Em::class,
+            ['class' => 'default-class'],
+        );
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\UIAwesome\Html\Core\Factory\SimpleFactory' in method 'testRenderWithGlobalDefaultsAreApplied'. (undefined)

(StaticAccess)